app/furniture/marketplace/marketplace_component.html.erb

```diff
@@ -1,5 +1,9 @@
 <div class="grid grid-cols-1 gap-6">
+
+
   <%= render delivery_area_component %>
 
+  <%= render Marketplace::MenuComponent.new(marketplace:, cart:) %>
```

Comment:
Should this be called `ProductList` or something that's not so semantically related with _food_? Also the domain concept of a "menu of products" and the UI concept of "menu of items" is a little confusing.

Reply:
I think I would prefer to lean more towards the `Marketplace` being a `CateringMarketplace` until we have other clients, rather than trying to make the underlying systems more generic.